src/utils/webhooksQueryOptions.tsx

```diff
@@ -0,0 +1,15 @@
+import { SpvWalletClientExtended } from '@/contexts';
+import { queryOptions } from '@tanstack/react-query';
+
+export interface WebhooksQueryOptions {
+  spvWalletClient: SpvWalletClientExtended;
+}
+
+export const webhooksQueryOptions = (opts: WebhooksQueryOptions) => {
+  const { spvWalletClient } = opts;
+
+  return queryOptions({
+    queryKey: ['webhooks', opts],
+    queryFn: async () => await spvWalletClient.AdminGetWebhooks(),
```

Comment:
`spvWalletClient.AdminGetWebhooks()` already returns a promise, the `async/await` pattern is unnecessary unless you plan to manipulate the result inside the function.
```suggestion
    queryFn: () => spvWalletClient.AdminGetWebhooks(),
```

Reply:
I've used the same approach in other queries. Also it helps to debug. However in this case let's do suggested change.